src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -17,7 +17,9 @@ internal sealed class Http3Connection : HttpConnectionBase, IDisposable
     {
         // TODO: once HTTP/3 is standardized, create APIs for these.
```

Comment:
Do we have an open issue tracking this?

Reply:
Yep, https://github.com/dotnet/runtime/issues/1293